_locales/_untranslated_en.json

```
@@ -79,5 +79,11 @@
   },
   "accounts_list_item_menu_label": {
     "message": "Actions with Profile"
+  },
+  "qrscan_ask_join_channel": {
+    "message": "Do you want to join the channel \"%1$s\"?"
+  },
+  "instant_onboarding_confirm_channel": {
+    "message": "Create a profile to join channel %1$s."
```

```suggestion
    "message": "Do you want to create a new profile and join the channel \"%1$s\"."
```

In line with

https://github.com/deltachat/deltachat-desktop/blob/63c2c938dd86976e7d3e94e19bd1243475fc70a9/_locales/en.xml#L635-L637

But it doesn't matter much, we'll pull the translations from upstream anyway.